app/Http/Controllers/Api/UsersController.php

```diff
@@ -246,6 +246,7 @@ public function index(Request $request)
                         'two_factor_optin',
                         'two_factor_enrolled',
                         'remote',
+                        'vipuser',
```

Comment:
This is a trivial thing, but I'd prefer if we use `vip` as the field name so it reads a little cleaner.